crates/egui/src/widgets/color_picker.rs

```diff
@@ -311,6 +339,49 @@ fn color_picker_hsvag_2d(ui: &mut Ui, hsva: &mut HsvaGamma, alpha: Alpha) {
     color_slider_2d(ui, v, s, |v, s| HsvaGamma { s, v, ..opaque }.into());
 }
 
+/// Shows 4 `DragValue` widgets to be used to edit the RGBA values.
+/// Alpha's `DragValue` is hidden when `Alpha::Opaque`.
+///
+/// Returns `Some([u8;4])` on change, with unmultiplied alpha values.
+fn srgba_edit_ui(ui: &mut Ui, rgba: [u8; 4], alpha: Alpha) -> Option<[u8; 4]> {
```

Comment:
Nit: Would it be possible for this to take a `&mut [u8; 4]` and return a `bool` instead?

Reply:
This suggestion makes the code slimmer and reduces the amount of variables to allocate. Committing.